AmmoDriver.md

```diff
@@ -0,0 +1,339 @@
+```
+
+See [examples/ammo.html](/examples/ammo.html) for a working sample.
+
+## Components
+
+### `ammo-body`
+
+An `ammo-body` component may be added to any entity in a scene. While having only an `ammo-body` will technically give you a valid physics body in the scene, only after adding an `ammo-shape` will your entity begin to collide with other objects.
+
+| Property                 | Default    | Description                                                                                                                                             |
+| ------------------------ |----------- | ------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| type                     | `dynamic`  | Options: `dynamic`, `static`, `kinematic`. See [ammo-body type](#ammo-body-type).                                                                       |
+| loadedEvent              | —          | Optional event to wait for before the body attempt to initialize.                                                                                       |
+| mass                     | `1`        | Simulated mass of the object, >= 0.                                                                                                                     |
+| gravity                  | `0 -9.8 0` | Set the gravity for this specific object.                                                                                                               |
+| linearDamping            | `0.01`     | Resistance to movement.                                                                                                                                 |
+| angularDamping           | `0.01`     | Resistance to rotation.                                                                                                                                 |
+| linearSleepingThreshold  | `1.6`      | Minimum movement cutoff before a body can enter `activationState: wants_deactivation`                                                                   |
+| angularSleepingThreshold | `2.5`      | Minimum rotation cutoff before a body can enter `activationState: wants_deactivation`                                                                   |
+| angularFactor            | `1 1 1`    | Constrains how much the body is allowed to rotate on an axis. E.g. `1 0 1` will prevent rotation around y axis.                                         |
+| activationState          | `active`   | Options: `active`, `island_sleeping`, `wants_deactivation`, `disable_deactivation`, `disable_simulation`. See [Activation States](#activation-states)   |
+| emitCollisionEvents      | `false`    | Set to true to enable firing of `collision` and `collision-end` events on this entity. See [Events](#events).                                           |
```

Comment:
To match A-Frame's event naming convention, maybe this should be `collisionstart` and `collisionend` or similar? The existing cannonjs events are called `collide`,  but they can't track start/stop states so I don't think it's necessary to match that.

Reply:
Done.